cssGenerators/getChapterHeaderCss.ts

```diff
@@ -119,15 +119,15 @@ export const getChapterHeaderCss = (
       word-break: break-word;
     }
 
-    ${prefixRule}.${addPrefix("chapter-title-card", prefix)} .${addPrefix("chapter-number", prefix)}{
+    ${prefixRule}.${addPrefix("chapter-title-card", prefix)} .${addPrefix("chapter-number", prefix)} span{
       color: ${
         renderLightHeaderText ? "white" : "black"
       };
       opacity: 1 !important;
       word-break: break-word;
     }
 
-    ${prefixRule}.${addPrefix("chapter-subtitle", prefix)} h3{
+    ${prefixRule}.${addPrefix("chapter-title-card", prefix)} .${addPrefix("chapter-subtitle", prefix)} h3{
```

Comment:
@ruveendra `${prefixRule}.` was there before for the `chapter-subtitle` and now it is missing on the second selector

Reply:
This is intentional i did this to fix an issue where light text is not getting applied for the subtitle and the chapter number. A similar implementation can be found in the chapter title. 
```
${prefixRule}.${addPrefix("chapter-title-card", prefix)} .${addPrefix("chapter-title", prefix)} h2{
      color: ${
        renderLightHeaderText ? "white" : "black"
      };
      opacity: 1 !important;
      word-break: break-word;
    }
```